local_groups/views.py

```diff
@@ -61,7 +61,7 @@ class GroupUpdateView(LoginRequiredMixin, UpdateView):
 
     # Redirect to same page on success
     def get_success_url(self):
-        return "/groups/" + self.object.slug + "/update/"
+        return "/groups/" + self.object.slug + "/manage/"
```

Comment:
I replaced this with a named mapping in my PR too. One of us will have to clean up the merge on this. I suggest just revert this line on your end to avoid merge conflict.

Reply:
https://github.com/Our-Revolution/site/pull/73/files#diff-7984a20a9d14ef5f869b494af09997c1R41